ega-data-api-dataedge/src/main/java/eu/elixir/ega/ebi/dataedge/rest/MetadataController.java

```diff
@@ -19,7 +19,10 @@
 import eu.elixir.ega.ebi.commons.config.VerifyMessage;
 import eu.elixir.ega.ebi.commons.shared.config.VerifyMessageNew;
 import eu.elixir.ega.ebi.commons.shared.dto.File;
+import eu.elixir.ega.ebi.dataedge.exception.ForbiddenException;
```

Comment:
@anandmohan777 is this an Exception that you can handle with the exciting 
`PermissionDenied | 403 | Authorization is required to access the resource` that we also use in Htsget? I'm up to reuse as much code as we have however, have a look if it will work for you or not :) But I think conceptually they are the same.